language/diem-vm/src/script_to_script_function.rs

```diff
@@ -680,41 +668,37 @@ pub(crate) fn remapping(script_bytes: &[u8]) -> Option<(&'static ModuleId, &'sta
 static ACCOUNT_ADMINISTRATION_SCRIPTS: Lazy<ModuleId> = Lazy::new(|| {
     ModuleId::new(
         CORE_CODE_ADDRESS,
-        Identifier::new("AccountAdministrationScripts").unwrap(),
+        ident_str!("AccountAdministrationScripts").to_owned(),
     )
 });
 
 static ACCOUNT_CREATION_SCRIPTS: Lazy<ModuleId> = Lazy::new(|| {
     ModuleId::new(
```

Comment:
I think this can be made const now?

Reply:
I don't think we can make `ModuleId::new` const b/c we can't make const `Identifier`s, as they need to allocate on the heap.

Reply:
Ah right... Unfortunate.